playback/src/main/kotlin/voice/playback/PlayerController.kt

```diff
@@ -132,6 +132,11 @@ class PlayerController
     it.seekTo((it.currentPosition - rewind.inWholeMilliseconds.coerceAtLeast(0)))
   }
 
+  fun pauseAtStart() = executeAfterPrepare {
+    it.pause()
+    it.seekTo(0)
```

Comment:
For files containing chapter marks this will seek to the beginning of a file and not to the beginning of a chapter mark

Reply:
I hope that's correct now, seek to the start time of the current chapter mark